js/private/image/index.ts

```diff
@@ -0,0 +1,120 @@
+import { Archiver, create } from 'archiver'
```

Comment:
is this still a workaround for some problem in rules_pkg with the pkg_tar rule? Or do we just not like their python dependency?

Reply:
python dependency is a big factor here. I don't feel comfortable adding another dependency to rules_js.

Reply:
okay that sounds right to me, could you just add some commenting or even a short README.md saying why it's designed this way?